packages/rocketchat-apps/client/admin/apps.js

```diff
@@ -1,38 +1,94 @@
 import { AppEvents } from '../communication';
 const ENABLED_STATUS = ['auto_enabled', 'manually_enabled'];
+const HOST = 'https://marketplace.rocket.chat';
 const enabled = ({ status }) => ENABLED_STATUS.includes(status);
 
-const sortByColumn = (array, column, inverted) => array.sort((a, b) => {
-	if (a[column] < b[column] && !inverted) {
-		return -1;
-	}
-	return 1;
-});
+const sortByColumn = (array, column, inverted) =>
+	array.sort((a, b) => {
+		if (a.latest[column] < b.latest[column] && !inverted) {
+			return -1;
+		}
+		return 1;
+	});
+
+const tagAlreadyInstalledApps = (instance, apps) => {
+	const installedApps = instance.installedApps.get() || [];
+	const installedIds = installedApps.map((app) => app.latest.id);
+
+	const tagged = apps.map((app) => {
+		console.log(app);
+		return {
+			latest: {
+				...app.latest,
+				_installed: installedIds.includes(app.latest.id),
+			},
+		};
+	});
+
+	return tagged;
+};
+
+const getApps = (instance) => {
+	instance.isLoading.set(true);
+
+	fetch(`${ HOST }/v1/apps?version=0.9.13`)
```

Comment:
Should probably pull version from somewhere.  I think #11835 alone will cause this to not really be accurate.  The required version should probably be somewhere that can be referenced.  Maybe RocketChatInfo?